issuer-list/src/main/java/com/adyen/checkout/issuerlist/IssuerListSpinnerView.kt

```diff
@@ -38,17 +40,30 @@ abstract class IssuerListSpinnerView<
         IssuerListComponentT>(context, attrs, defStyleAttr),
     AdapterView.OnItemSelectedListener {
 
-    private lateinit var issuersSpinner: AppCompatSpinner
-    private lateinit var issuersAdapter: IssuerListSpinnerAdapter
+    protected val termsAndConditionsTextView: TextView?
+        get() = rootView.findViewById<TextView>(R.id.textview_termsAndConditions)
+    private val autoCompleteTextViewIssuers: AutoCompleteTextView
+        get() = rootView.findViewById(R.id.autoCompleteTextView_issuers)
+    private val textInputLayoutIssuers: TextInputLayout
+        get() = rootView.findViewById(R.id.textInputLayout_issuers)
+
+    private val issuersAdapter: SimpleTextListAdapter<IssuerModel> = SimpleTextListAdapter(context)
 
     init {
+        orientation = VERTICAL
+        layoutParams = LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.WRAP_CONTENT)
         LayoutInflater.from(getContext()).inflate(R.layout.issuer_list_spinner_view, this, true)
     }
 
     override fun initView() {
-        issuersSpinner = findViewById<AppCompatSpinner?>(R.id.spinner_issuers).apply {
-            adapter = issuersAdapter
-            onItemSelectedListener = this@IssuerListSpinnerView
+        autoCompleteTextViewIssuers.apply {
+            inputType = 0
+            setAdapter(issuersAdapter)
+            onItemClickListener = AdapterView.OnItemClickListener { _, _, position, _ ->
+                d(TAG, "onItemSelected - " + issuersAdapter.getItem(position).name)
```

Comment:
If functions do not have a descriptive name I would not import them directly. What is `d()`? No idea. `Logger.d()`? Ohhh debug data is being logged

Reply:
I guess this happen because of copy and paste and android studio decided it will be like that 😅

Reply:
Haha yeah sometimes it does that